sds_data_manager/stacks/data_storage_stack.py

```diff
@@ -0,0 +1,39 @@
+"""Data Storage Stack"""
+from aws_cdk import Environment, RemovalPolicy, Stack
+from aws_cdk import aws_s3 as s3
+from constructs import Construct
+
+
+class DataStorageStack(Stack):
+    """Data Storage Resources"""
+
+    def __init__(self,
+                 scope: Construct,
+                 construct_id: str,
+                 sds_id: str,
+                 env: Environment,
+                 **kwargs) -> None:
+        """DataStorageStack constructor
+
+        Parameters
+        ----------
+        scope : App
+        construct_id : str
+        sds_id : str
+            Name suffix for stack
+        env : Environment
+            Account and region
+        """
+        super().__init__(scope, construct_id, env=env, **kwargs)
+
+        #TODO: removal policy may change
+        removal_policy = RemovalPolicy.DESTROY
+        self.bucket_name = f"archive-{sds_id}"
+
+        # Bucket for processed data
+        self.archive_bucket = s3.Bucket(self, f"ArchiveBucket-{sds_id}",
```

Comment:
What's the difference between this bucket and the data bucket we currently have? I thought all processed data was going to go to the data bucket.

Reply:
They are the same. I will import that bucket. Sorry that's confusing.